docs/learn/use-netlify-cms/use-netlify-cms.md

```diff
@@ -0,0 +1,202 @@
+  name: git-gateway
+  branch: main # Branch to update (optional; defaults to main)
+collections:
+  - name: 'blog' # Used in routes, e.g., /admin/collections/blog
+    label: 'Blog' # Used in the UI
+    folder: 'blog' # The path to the folder where the documents are stored
+    create: true # Allow users to create new documents in this collection
+    slug: '{{year}}-{{month}}-{{day}}-{{slug}}' # Filename template, e.g., YYYY-MM-DD-title.md
+    fields: # The fields for each document, usually in front matter
+      - { label: 'Layout', name: 'layout', widget: 'hidden', default: 'blog' }
+      - { label: 'Title', name: 'title', widget: 'string' }
+      - { label: 'Description', name: 'description', widget: 'string' }
+      - { label: 'Slug', name: 'slug', widget: 'string' }
+      - { label: 'Publish Date', name: 'publishDate', widget: 'datetime' }
+      - { label: 'Body', name: 'body', widget: 'markdown' }
+      - {
+          label: 'Published',
+          name: 'published',
+          widget: 'boolean',
+          default: true,
+        }
+media_folder: 'static/img' # Folder where user uploaded files should go
+```
+
+You can put this file anywhere you would like, as long as it's included in the output after the site is generated with Scully. One possibility would be to add it to the `src` folder, and then include it in the `assets` array for the project. This would handle the `config.yml` just like the favicon is handled.
+
+```json
+<!-- angular.json -->
+...
+  "assets": ["src/favicon.ico", "src/config.yml", "src/assets"],
+...
+```
+
+When the application is built, the config file is copied to the `dist/static` folder. Once you have this `config.yml` file in the correct place after the app is built, you are good to move on to the next step. Commit all your changes and push them to the `main` branch of your repo, which will trigger a new build and deploy.
+
+## [Accessing the CMS from the Browser](#accessing-the-cms-from-the-browser)
+
+Once your app has been built and deployed by Netlify, you can visit the site and go to the /admin route. When the script is injected, Netlify should take over and ask you to authenticate with Netlify Identity. After you authenticate, you should be taken to a CMS landing page. It will look something like this:
+
+![Netlify CMS Landing Page](/assets/images/blog-posts/scully-netlify-cms/netlify-cms-landing-page.png)
+
+From here, you will be able to add and edit new blog posts in a user friendly rich text or markdown editor. You won't have to edit the markdown blog post files directly any more.
+
+## [Conclusion](#conclusion)
+
+Netlify CMS is a great tool for placing a lightweight CMS on top of your Scully app. All the code can live in one codebase, and everything is just backed by markdown files directly. It doesn't take much to add it to your site, and is cost effective on top of all of that.
+
+You can see an example of all this in [this GitHub repo](https://github.com/pjlamb12/scully-netlify-cms).
```

Comment:
I'm not sure if we should link to an external resource like this?
@aaronfrost your thoughts?

Reply:
Just wanted to follow up on this and see what you guys think?

Reply:
As @aaronfrost did react on this yet, I would say take it out for now. If he is ok with it we can put it back in.

Reply:
I got that external link removed